README.md

```diff
@@ -75,19 +69,14 @@ risk = evaluator.risk()
 ## Getting started
```

Comment:
I'd put “Getting started” just after “Setup and installation”. In fact, the “Basic usage pattern” is in the notebook, too.